app/googleanalytics/pixels.py

```diff
@@ -1,35 +1,51 @@
-from urllib.parse import urlencode
+import os
 
 from flask import current_app
 
 
-def build_ga_pixel_url(
-    notification,
-    provider,
-):
-    url_params_dict = {
-        'v': '1',
-        't': 'event',
-        'tid': current_app.config['GOOGLE_ANALYTICS_TID'],
-        'cid': notification.id,
-        'aip': '1',
-        'ec': 'email',
-        'ea': 'open',
-        'el': notification.template.name,
-        'dp': f"/email/vanotify"
-        f"{'/' + notification.service.organisation.name if notification.service.organisation else ''}"
-        f"/{notification.service.name}"
-        f"/{notification.template.name}",
-        'dt': notification.subject,
-        'cn': notification.template.name,
-        'cs': provider.get_name(),
-        'cm': 'email',
-        'ci': notification.template.id,
+TEST_DOMAIN = 'https://test-api.va.gov/notify/'
+DEV_DOMAIN = 'https://dev-api.va.gov/notify/'
+PERF_DOMAIN = 'https://sandbox-api.va.gov/notify/'
+STAGING_DOMAIN = 'https://staging-api.va.gov/notify/'
+PROD_DOMAIN = 'https://api.va.gov/notify/'
+
+NOTIFICATION_API_GA4_GET_ENDPOINT = 'ga4/open-email-tracking'
```

Comment:
After reviewing some unit tests and docs, I can't use the `for_url()`  method outside the context of a request. This means I can't get the new GET endpoint url programmatically. I have opts to save the path as a constant